src/app/pages/billing/billing.component.ts

```diff
@@ -23,27 +22,29 @@ import { UserService } from '../../services/user.service';
   styleUrl: './billing.component.scss',
 })
 export class BillingComponent implements OnInit {
-  exchangeData: ExchangeData[] = [];
-  userBalances: UserBalance[] = [];
-  userBalanceColumns: string[] = ['currency', 'amount'];
-  exchangeDataColumns: string[] = ['currency', 'rate', 'date'];
-  isLoading = false;
-  error = '';
+  public exchangeData: ExchangeData[] = [];
+  public userBalances: UserBalance[] = [];
+  public userBalanceColumns: string[] = ['currency', 'amount'];
+  public exchangeDataColumns: string[] = ['currency', 'rate', 'date'];
+  public isLoading = false;
+  public error = '';
 
   private billService = inject(BillService);
   private destroyRef = inject(DestroyRef);
-  private userService = inject(UserService);
 
-  ngOnInit(): void {
-    const user = this.userService.getUser();
+  public ngOnInit(): void {
+    this.loadAccountData();
+  }
 
-    if (!user) {
-      return;
-    }
+  public updateAccountData(): void {
```

Comment:
Why did you create this method? For just calling another method, why no just call method that need?